src/components/Menu.tsx

```diff
@@ -1,16 +1,73 @@
+import Button from '@material-ui/core/Button';
+import Container from '@material-ui/core/Container';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+
 import React from 'react';
 import { Link } from 'react-router-dom';
 
+const OuterContainer = withStyles({
+  root: {
+    padding: '3rem 2rem',
+    textAlign: 'center',
+    height: '100vh',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+})(Container);
+
+const InnerContainer = withStyles({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: 'fit-content',
+  },
+})(Container);
+
+const StyledButton = withStyles({
+  root: {
+    marginBottom: '2rem',
+  },
+})(Button);
+
+const TransparentButton = withStyles({
+  root: {
+    color: '#009688',
+    background: 'none',
+    '&:hover': {
+      background: 'none',
+    },
+  },
+})(Button);
+
+const styledButtonLink = {
+  textDecoration: 'none',
+  color: 'white',
+};
+
+const transparentButtonLink = {
+  textDecoration: 'none',
+  color: '#009688',
+};
+
 export default function Menu() {
   return (
-    <div>
-      <h2>PODZIELNIA LOGO</h2>
-      <button>
-        <Link to={'/quiz'}>Nowa gra</Link>
-      </button>
-      <button>
-        <Link to={'/about'}>O projekcie</Link>
-      </button>
-    </div>
+    <OuterContainer>
+      <Typography variant="h4">PODZIELNIA LOGO</Typography>
+      <InnerContainer>
+        <StyledButton>
+          <Link to={'/quiz'} style={styledButtonLink}>
```

Comment:
Remember to import routes
```suggestion
          <Link to={routes.quiz} style={styledButtonLink}>
```